test/checks/landmarks/landmark-is-unique-after.js

```diff
@@ -0,0 +1,84 @@
+describe('landmark-is-unique-after', function() {
+	'use strict';
+
+	var checkContext = axe.testUtils.MockCheckContext();
+	function createResult(result, data) {
+		return {
+			result: result,
+			data: data
+		};
+	}
+
+	function createResultWithSameRelatedNodes(result, data) {
+		return {
+			...createResult(result, data),
```

Comment:
Babel isn't enabled in our tests, so we need to restrict it to ES3 features. This isn't failing because Appveyor is currently disabled, a fix for that landed recently. There's an issue open to enable Babel on our tests, but for now, use `Object.assign`.

Reply:
Done.